addons/website_event/tests/test_event_internals.py

```diff
@@ -25,10 +25,12 @@ def test_event_type_configuration_from_type(self):
 
         self.assertEqual(
             event.question_ids.mapped('question_type'),
-            ['simple_choice', 'simple_choice', 'text_box'])
-        self.assertEqual(event.specific_question_ids.title, 'Question1')
+            ['name', 'email', 'phone', 'simple_choice', 'simple_choice', 'text_box'])
+        self.assertEqual(event.specific_question_ids.filtered(
```

Comment:
You could also adapt to test the title of the added questions (name / phone / email) ?

Reply:
Up?

Reply:
The test is a few lines below ^^